testing_and_setup/compass/ocean/hurricane/hurricane_wind_pressure/plot_winds_on_mpaso_mesh.py

```diff
@@ -0,0 +1,50 @@
+import netCDF4
+import matplotlib.pyplot as plt
+import numpy as np
+from mpl_toolkits.basemap import Basemap
+plt.switch_backend('agg')
```

Comment:
@sbrus89, can we please fix basemap reference here?  This needs to be a new commit off the initial import.

Reply:
I can also do this too, which might make more sense given that I'm diving in to understand here.

Reply:
Let's discuss.

Reply:
Really good catch. I'll fix this.